collectionfunctions.php

```diff
@@ -83,7 +93,36 @@ function checkNew(PDO $database) {
                 header('Location: collectionchallenge.php');
             }
         } else {
-            echo "<div class='error'>Unable to add Pokemon to the collection</div>";
+            echo "<div class='error'>Unable to add Pokémon to the collection</div>";
+        }
+    }
+}
+
+function editPoke(PDO $database) {
+    if (isset($_POST['name'], $_POST['type1'], $_POST['hp'], $_POST['attack'], $_POST['defense'], $_POST['spAttack'], $_POST['spDefense'], $_POST['speed'])) {
```

Comment:
Could potentially put these 2 lines in a function so you aren't repeating yourself (also in the checkNew function above)? :)

Reply:
Amended to a variable :)